hazelcast/src/main/java/com/hazelcast/internal/partition/operation/BaseMigrationSourceOperation.java

```diff
@@ -35,8 +35,6 @@
 
 abstract class BaseMigrationSourceOperation extends BaseMigrationOperation {
 
-    boolean returnResponse = true;
```

Comment:
No need for this any longer since the MigrationRequestOperation makes use of CallStatus to indicate if there is a response or not.